csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -33,7 +33,8 @@ private predicate canCreateCompilerGeneratedElement(Element generatedBy, int nth
   generatedBy instanceof DelegateCreation and
   nth in [0 .. DelegateElements::noGeneratedElements(generatedBy)]
   or
-  generatedBy instanceof DelegateCall and nth in [0 .. DelegateElements::noGeneratedElements(generatedBy)]
+  generatedBy instanceof DelegateCall and
+  nth in [0 .. DelegateElements::noGeneratedElements(generatedBy)]
```

Comment:
(Also in the above)

Reply:
`noGeneratedElements(generatedBy)` was the correct number, but I realize it is a bit confusing. I have made some changes so that now ` ::noGeneratedElements(generatedBy) - 1` is the correct number.

Reply:
`noGeneratedElements(generatedBy)` was the correct number, but I realize it is a bit confusing. I have made some changes so that now ` ::noGeneratedElements(generatedBy) - 1` is the correct number.

Reply:
`noGeneratedElements(generatedBy)` was the correct number, but I realize it is a bit confusing. I have made some changes so that now ` ::noGeneratedElements(generatedBy) - 1` is the correct number.